File: javascript/src/providers/EmailTemp.js
import { Emailfake } from './Emailfake.js';

/**
 * email-temp.com provider — same gasmurl backend as emailfake.com.
 * Only the base URL and provider name differ; all logic is inherited.
 */
export class EmailTemp extends Emailfake {
  /** @param {{ randomUA?: boolean, useCookies?: boolean }|undefined} opts */
  constructor(opts = {}) {
    super(opts, 'email-temp', 'https://email-temp.com');
  }
}

File: go/providers/email_temp.go
package providers

// EmailTemp is the email-temp.com provider (same gasmurl backend as emailfake).
type EmailTemp struct {
	*gasmurlBackend
}

// NewEmailTemp creates a new EmailTemp provider.
func NewEmailTemp(config map[string]string) *EmailTemp {
	return &EmailTemp{newGasmurlBackend(config, "email-temp", "https://email-temp.com", ".email-temp.com")}
}
